hearth/lib/hearth/http/header_list_builder.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Http
```

Comment:
I'm pretty sure everything in this module uses HTTP in all caps. I'm noticing both parser and builder do not do this! Please change the namespace of both of these.